src/usr/local/www/system_routes_edit.php

```diff
@@ -199,8 +199,8 @@
 				foreach ($delete_targets as $dts) {
 					if (is_ipaddrv6($dts)) {
 						$family = "-inet6";
-					}
```

Comment:
Why was this removed? Looks like it would introduce a syntax error here.

Reply:
typo, fixed